src/test/shell/bazel/run_test.sh

```diff
@@ -94,3 +94,27 @@ eof
   bazel run --enable_runfiles=no $pkg:x >&$TEST_log || fail "expected success"
   bazel run --enable_runfiles=yes $pkg:x >&$TEST_log || fail "expected success"
 }
+
+function test_windows_argument_escaping() {
+  if ! "$is_windows"; then
+    return # Run test only on Windows.
+  fi
+
+  local -r pkg="pkg${LINENO}"
+  mkdir $pkg
+  cat >$pkg/BUILD <<eof
+sh_binary(
+  name = "a",
+  srcs = [":a.sh"],
+)
+eof
+  cat >$pkg/a.sh <<eof
+echo Hello World
+eof
+  disable_errexit
+  output="$(BAZEL_SH="C:/first_part second_part" bazel run --run_under=":;" $pkg:a 2>&1)"
```

Comment:
Add a comment explaining why this run_under value was chosen and what the expected result of this command is.

The details in the PR are useful, but should be in the actual test for future readers.

Reply:
Done.

Reply:
The value of `--run_under` turns out to be irrelevant, let me know if you still want this bit of information in the comment.